packages/tc-api-rest-handlers/lib/relationships/write.js

```diff
@@ -49,22 +49,40 @@ WHERE related.code IN $${key}
 `;
 };
 
-const addPropsToQueries = (relationshipPropQueries, value) => {
-	// value: { code: 'node code', someString: 'some string'...}
-	Object.entries(value).forEach(([k, v]) => {
-		if (k !== 'code') {
-			// If no node matches the CASE expression, the expression returns a null.
-			// and no action will be taken
+const addPropsToQueries = (
```

Comment:
This looks like it's probably good :smile: 

but it does need tests in `api-rest-handlers/__tests__`

Need to check that
- a multiple choice property can be written to a relationship
- a boolean property can be written to a relationship
- different values can be written to 2 relationships at the same time
- (maybe) different values can be written to 2 relationships _of different types_ at the same time (though I'm not sure if the schema defines any relationships that will allow us to test this)